test/corpus/expressions.txt

```diff
@@ -850,8 +866,8 @@ class C {
     a
       .b
       // comment1
-      /* 
-         comment2 
+      /*
```

Comment:
Changes here and below are caused by changes in behavior of `tree-sitter test -u`, which now removes trailing spaces